frontend/src/concepts/hardwareProfiles/useServingHardwareProfileConfig.ts

```diff
@@ -5,9 +5,9 @@ import {
   UseHardwareProfileConfigResult,
 } from './useHardwareProfileConfig';
 
-const useServingHardwareProfileConfig = (
+export const extractHardwareProfileConfigFromInferenceService = (
   inferenceService?: InferenceServiceKind | null,
-): UseHardwareProfileConfigResult => {
+): Parameters<typeof useHardwareProfileConfig> => {
   const legacyName =
     inferenceService?.metadata.annotations?.['opendatahub.io/legacy-hardware-profile-name'];
```

Comment:
Legacy profiles will be deprecated in version 3.0, this can however be addressed as part of the broader cleanup in [RHOAIENG-32955](https://issues.redhat.com/browse/RHOAIENG-32955)

Reply:
Very nice, thank you